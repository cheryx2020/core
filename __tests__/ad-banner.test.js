import React from "react";
import renderer from "react-test-renderer";
import AdBanner from "../src/components/ad-banner/ad-banner";

test("renders AdBanner component correctly", () => {
const component = renderer.create(<AdBanner />);
const tree = component.toJSON();
expect(tree).toMatchSnapshot();
});