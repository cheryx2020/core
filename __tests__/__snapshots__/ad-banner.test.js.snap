// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders AdBanner component correctly 1`] = `
<ins
  className="adsbygoogle adbanner-customize"
  data-ad-client="ca-pub-4179656549806780"
  data-ad-format="fluid"
  data-ad-layout="in-article"
  data-ad-slot="9675079770"
  style={
    {
      "display": "block",
    }
  }
/>
`;
